.github/workflows/main.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        os: [centos7, debian10, ubuntu1804, ubuntu2004]
-        pg: [pg11, pg12]
+        os: [ubuntu2004, ubuntu1804, debian10, centos7]
+        pg: [pg11, pg12, pg14, pg13]
```

Comment:
Also does this order have any significance?

Reply:
No I removed the entries and re-added them when I  was working on the OS . I can revert that line